package.json

```diff
@@ -144,7 +144,8 @@
     "typescript": "3.5.3"
   },
   "resolutions": {
-    "@types/react": "16.9.1"
+    "@types/react": "16.9.1",
+    "react": "16.8.6"
```

Comment:
Otherwise react-intl among others due to loose ranges keep pulling in v16.9 which leaves us with two versions and breaks out tests as we throw when we log.